lib/ae_mdw/names.ex

```diff
@@ -70,6 +70,16 @@ defmodule AeMdw.Names do
   @min_bin Util.min_bin()
   @max_bin Util.max_256bit_bin()
 
+  @spec count_names(state(), query()) :: non_neg_integer()
+  def count_names(state, query) do
+    with {:ok, filters} <- Util.convert_params(query, &convert_param/1) do
+      filters
+      |> build_height_streamer(state, nil, nil, nil)
+      |> then(fn streamer -> streamer.(:backward) end)
```

Comment:
Is this going to go through all of the names? There's no limit to how many names there could be, this could take a lot of time

Reply:
It is a slow query yeah, but the idea is so that this can be used to get all active names owned by an account. The problem with counting the results from the regular endpoint is that it's paginated and can't be counted